### PR TITLE
Make *Workspace::Output return type non-const ref

### PR DESCRIPTION
**Pull request by Kh4L**
Signed-off-by: Serge Panev <spanev@nvidia.com>

### CODE REVIEW

**Comment by Kh4L** [comment]
Build 611481

**Comment by Kh4L** [comment]
Build 612212

**Comment by Kh4L** [comment]
Build 612321

File: dali/pipeline/operators/resize/new_resize.cu [hunk 1]
```diff
@@ -186,23 +186,23 @@ bool DataDependentSetupGPU(const TensorList<GPUBackend> &input, TensorList<GPUBa
...
+  output.set_type(input.type());
 
   CollectPointersForExecution(reshapeBatch ? 1 : batch_size, input, inPtrs, output, outPtrs);
   return newResize;
 }
 
 void CollectPointersForExecution(size_t batch_size,
                                  const TensorList<GPUBackend> &input, vector<const uint8 *> *inPtrs,
-                                 TensorList<GPUBackend> *output, vector<uint8 *> *outPtrs) {
+                                 TensorList<GPUBackend> &output, vector<uint8 *> *outPtrs) {
```
Comment (mzient) [inline dali/pipeline/operators/resize/new_resize.cu]: When you're at it - why not change this vector to a reference, too?

**Pull request merged**